beacond/cmd/types.go

```diff
@@ -114,9 +105,6 @@ type (
 	// CometBFTService is a type alias for the CometBFT service.
 	CometBFTService = cometbft.Service[*Logger]
 
-	// ConsensusMiddleware is a type alias for the consensus middleware.
-	ConsensusMiddleware = cometbft.MiddlewareI
-
```

Comment:
I believe unnecessary once Middleware has been encapsulated in consensus service